src/uuid/builder/uuidutils.rs

```diff
@@ -156,3 +165,34 @@ impl UuidUtils {
         }
     }
 }
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum Version {
+    /// An unknown version.
+    VersionUnknown = 0,
```

Comment:
Minor comment about naming convention. 
The variant name usually does not repeat the enum name.
E.g.: `Version::Unknown` instead of `Version::VersionUnknown`.

Reply:
done